.github/CODEOWNERS

```diff
@@ -0,0 +1,9 @@
+#
+# Assigning ownership of project modules
+# https://help.github.com/articles/about-codeowners/
+#
+
+# Default code owners, matches here for code review if nothing else is described.
+*	@benwerd @erinjo @mapkyca
```

Comment:
Erin shouldn't be a part of this for the time being, as she's not able to approve / commit code.